### ISSUE TITLE
bin/ots init fails

### ISSUE THREAD

**Issue by MikeO3** [onetimesecret/onetimesecret#199]
This is the same as #165 however later versions.
Ubuntu 22.04
bundle install completes

```
ots@onetimesecret:~/onetimesecret$ bin/ots init
/var/lib/gems/3.0.0/gems/storable-0.8.9/lib/proc_source.rb:41:in `<top (required)>': uninitialized constant RubyToken (NameError)
Did you mean?  RubyLex
        from /var/lib/gems/3.0.0/gems/storable-0.8.9/lib/storable.rb:35:in `require'
        from /var/lib/gems/3.0.0/gems/storable-0.8.9/lib/storable.rb:35:in `<class:Storable>'
        from /var/lib/gems/3.0.0/gems/storable-0.8.9/lib/storable.rb:33:in `<top (required)>'
        from /var/lib/gems/3.0.0/gems/sysinfo-0.7.3/lib/sysinfo.rb:2:in `require'
        from /var/lib/gems/3.0.0/gems/sysinfo-0.7.3/lib/sysinfo.rb:2:in `<top (required)>'
        from /home/ots/onetimesecret/lib/onetime.rb:13:in `require'
        from /home/ots/onetimesecret/lib/onetime.rb:13:in `<top (required)>'
        from <internal:/usr/lib/ruby/vendor_ruby/rubygems/core_ext/kernel_require.rb>:85:in `require'
        from <internal:/usr/lib/ruby/vendor_ruby/rubygems/core_ext/kernel_require.rb>:85:in `require'
        from bin/ots:9:in `<main>'
ots@onetimesecret:~/onetimesecret$

```

**Comment by vinay-chintalapati** [comment]
I have the same issue as well

**Comment by River2906** [comment]
Yep - same here

**Comment by delano** [comment]
This issue has been resolved as of [v0.13.0](https://github.com/onetimesecret/onetimesecret/releases/tag/latest). Also available via the official docker images now. 

https://hub.docker.com/r/onetimesecret/onetimesecret/tags
